src/averageAges.js

```diff
@@ -14,12 +14,30 @@
  *
  * @return {number}
  */
+
+function sexDefined(value) {
+  return person => person.sex === value;
+}
+
+function diedInCentury(century) {
+  return person => century ? Math.ceil(person.died / 100) === century : person;
+}
+
+function toListAges() {
+  return person => person.died - person.born;
+}
```

Comment:
Краще створити функцію, яка приймає person і повертає різницю.
Створювати фабрику функцій тут непотрібно
```suggestion
const toListAge = (person) => person.died - person.born
```

Reply:
А чим це краще?